tests/whist/server/routes/test_index.py

```diff
@@ -15,4 +15,4 @@ def test_read_index(self):
         """
         response = self.client.get('/')
         self.assertEqual(response.status_code, 200)
-        self.assertDictEqual(response.json(), {'game': 'whist'})
+        self.assertDictEqual(response.json(), {'info': {'game': 'whist', 'version': '0.1.0rc2'}})
```

Comment:
can we not have that hardcoded in the test? so we dont have to change 10 different files to increment a version.

Reply:
Yes I was thinking about this. Will do this later

Reply:
The whist core version is not hard coded any more.